src/flamegraph/color/palette_map.rs

```diff
@@ -9,7 +9,7 @@ use std::str::FromStr;
 
 /// Mapping of the association between a function name and the color used when drawing information
 /// from this function.
-#[derive(Default)]
+#[derive(Debug, Default)]
```

Comment:
Let's also add `Clone` to this. Seems pretty reasonable that that might be something someone wants to do. Perhaps also `Eq, PartialEq`.